<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_reservation</class>
 <widget class="QMainWindow" name="gestion_reservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Reservations</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>911</width>
        <height>641</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>GÃ©rer Reservation</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>911</width>
          <height>671</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>12</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>prenom</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_id_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>10</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nom_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_prenom_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>90</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="on_pb_ajouter_clicked">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>290</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>nombre</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>260</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>id_service</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>date_d</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>date_f</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_idservice">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>250</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nombre">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>130</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_date_d">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>170</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_date_f">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>210</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTableView" name="tablreservation_2">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>10</y>
           <width>541</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>330</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Tri Croissant</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>330</y>
           <width>111</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Tri Decroissant</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>371</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>recherche</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_rech"/>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="on_pb_actualiser">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>370</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Actualiser</string>
         </property>
        </widget>
        <widget class="QPushButton" name="on_pb_supprimer">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>410</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="on_pb_modifier">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>330</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>370</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Statistiques</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_reser">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>370</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="on_pb_choisir">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>370</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>choisir</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>330</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>imprimer</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Services</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>531</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>GÃ©rer Service</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_type">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_disponibilite">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID_service">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>19</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>59</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>disponibilite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>19</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Service</string>
        </property>
       </widget>
       <widget class="QPushButton" name="on_pb_ajouter_service_clicked">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>160</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="on_pb_trier_service_croissante">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>270</y>
          <width>201</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri Croissant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_rech_2">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>300</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_service">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>0</y>
          <width>451</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="on_pb_actualiser_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>300</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Actuialiser</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>300</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="on_pb_trier_service_decroissante">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>270</y>
          <width>141</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri Decroissant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="on_pb_modifier_service">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_service">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>240</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="on_pb_supprimer_service">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>280</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>240</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>choisir</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
